### PR TITLE
Add test verifying VPA doesn't block patches

### PR DESCRIPTION
**Pull request by bskiba**

### CODE REVIEW

**Comment by bskiba** [comment]
@kgolab

File: vertical-pod-autoscaler/e2e/v1beta1/common.go [hunk 1]
```diff
@@ -198,15 +199,48 @@ func NewVPA(f *framework.Framework, name string, selector *metav1.LabelSelector)
...
-	vpaClientSet := vpa_clientset.NewForConfigOrDie(config)
-	vpaClient := vpaClientSet.AutoscalingV1beta1()
-	_, err = vpaClient.VerticalPodAutoscalers(ns).Create(vpa)
+	vpaClientSet := GetVpaClientSet(f)
+	_, err := vpaClientSet.AutoscalingV1beta1().VerticalPodAutoscalers(f.Namespace.Name).Create(vpa)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred(), "unexpected error creating VPA")
+}
+
+func PatchVpaRecommendation(f *framework.Framework, vpa *vpa_types.VerticalPodAutoscaler,
+	recommendation *vpa_types.RecommendedPodResources) {
```
Comment (jbartosik) [inline vertical-pod-autoscaler/e2e/v1beta1/common.go]: Nit: This line break looks a bit weird.

**Comment by jbartosik** [comment]
Please fix lint error.

**Comment by bskiba** [comment]
Fixed.

**Pull request merged**